### ISSUE TITLE
importing 3D images 

### ISSUE THREAD

**Issue by UlrichThomsen** [wonderunit/storyboarder#2038]
on youtube there's a super fast tutorial on how to import a 3D model in SketchUp, then import into Blender in order to make a 3D model you can import in Storyboader shot  generator. 
The Blender thing is lightening fast and I'm lost. Also, there's a new blender version that I have. I tried to stop and go, and I can get something imported in Shot generator, but it's a weird image that has nothing to do with the original image I got on 3D Warehouse.  

my Blender is not doing the same things as the super fast tutorial. also it's newer version.  
does someone have a more painting my numbers tutorial from 3DWarehouse to SketchUp to finished render in Blender, with the newest software? 

Best Ulrich

**Comment by audionerd** [comment]
Which YouTube video were you working from?

There are some guides in the wiki which might be helpful:

https://github.com/wonderunit/storyboarder/wiki/Exporting-Objects-from-Sketchup

https://github.com/wonderunit/storyboarder/wiki/Creating-custom-3D-Models-for-Shot-Generator

**Comment by UlrichThomsen** [comment]
The first link. 

It clicks around it’s so hard to follow. (Being new to all this obviously) 

Second link I haven’t tried. Which is not what I’m after water. Just after some readymade objects to add to shot generator.  

Med venlig hilsen / Best Regards, 

Ulrich Thomsen

PRO TEMPORE FILM ApS

**Comment by audionerd** [comment]
> Just after some readymade objects to add to shot generator.

Have you seen this? https://github.com/wonderunit/shot-generator-models

**Comment by UlrichThomsen** [comment]
That’s fine. Some do freeze up my system though, BUT these re only a fragment of what 3d warehouse offers.  

Do you know how to explain the blender video at a slower speed and take it by the numbers? 

Med venlig hilsen / Best Regards, 

Ulrich Thomsen

PRO TEMPORE FILM ApS

**Comment by UlrichThomsen** [comment]
Also is there a way to install those add ons and not just import them as you go 

Med venlig hilsen / Best Regards, 

Ulrich Thomsen

PRO TEMPORE FILM ApS

**Comment by UlrichThomsen** [comment]
now I have, but very limited to what 3D warehouse offers. some freeze up my system also.
